response.go

```diff
@@ -86,13 +95,15 @@ func (body *Body) SetFile(path string) error {
 	return nil
 }
 
+// Response is the the preconfigured HTTP response that will be returned to the client.
 type Response struct {
 	Status  int
 	Headers http.Header
 	Body    Body
 	Delay   time.Duration
 }
 
+// UnmarshalJSON inflats the Response from []byte representing JSON.
```

Comment:
what does `inflats` means :)

Reply:
Meant to be 'inflates' as a synonym to unmarshalling/deserialization.